src/middlewared/middlewared/plugins/network.py

```diff
@@ -1491,6 +1486,38 @@ def choices(self, options):
             'exclude': ['bridge', 'epair', 'tap', 'vnet', 'vlan'],
         })
 
+    @private
+    def scan_vrrp(self, ifname, count=10, timeout=10):
+        # This runs tcpdump looking for VRRP packets.  If
+        # none are seen we have a gun in the glovebox that times the
+        # tcpdump out after `timeout` seconds. If we do see VRRP packets we stop
+        # after seeing `count` of them. If there were more than `count` VRRP
+        # devices on the broadcast domain we wouldn't get them all.
+        proc = subprocess.Popen(
+            f"tcpdump -l -n {f'-c {count}' if count else ''} -i {ifname} vrrp 2> /dev/null"
```

Comment:
This will add an extra space between `-n` and `-i` if count is false but i don't think that would hurt ?

Reply:
Its shell, so not a problem, looked better than the alternative of not having any spaces between the f-string and other param ; )